benchexec/tablegenerator/react-table/src/components/SelectColumn.js

```diff
@@ -181,6 +181,40 @@ export default class SelectColumn extends React.Component {
     this.setState({ list });
   };
 
+  backButton = () => {
+    // event listener for back button
+    (function (window) {
+      window.addEventListener("hashchange", function (e) {
+        console.log("back button clicked");
+      });
+    })(window);
+
+    // Change the function of the browser's back button
+    (function (window, location) {
+      window.history.replaceState(
+        null,
+        document.title,
+        location.pathname + "#!/stealingyourhistory",
```

Comment:
What kind of string is `stealingyourhistory`?!? We certainly do not want users to see this.